### ISSUE TITLE
reCaptcha Challenge requested execute

### ISSUE THREAD

**Issue by niuyaweiqq** [TeamNewPipe/NewPipe#4282]
I found a problem. When my VPN switched to France, the song could not be downloaded and it prompted: reCaptcha Challenge requested execute. I check the detailed error information: https://www.youtube.com/watch?v=phpGv04wx5c&pbj=1
phpGv04wx5cmp3. But I can open this music through webview, and it can also be downloaded normally in VPNs of other countries. Please check whether the error message needs to be changed, or there are other reasons.

**Comment by opusforlife2** [comment]
Do you get a reCAPTCHA to solve or just see the error?

**Comment by niuyaweiqq** [comment]
> 您得到reCAPTCHA解决或只是看到错误？
I just saw this error. At first, I suspected that the playback was restricted in France, but my webview can be played normally, but the entire webpage cannot be downloaded. The complete error message is: java.io.FileNotFoundException: https://www.youtube. com/watch?v=phpGv04wx5c&pbj=1

**Comment by opusforlife2** [comment]
Alright, can you try this?

- Export database from settings
- Clear app data
- Import database, but when it asks to restore settings, say no. Restoring settings will also restore cookies and we don't want that.
- Try that URL again.

After you're done testing, you can clear app data and import the database along with settings to get back to where you started.

**Comment by niuyaweiqq** [comment]
The problem still exists. I tried to switch VPN downloads in other countries without any problems. My question is whether this music is forbidden in France, if it is forbidden, why can webview be played, and if it is not forbidden in France, why is there only a French VPN Can't download the webpage. Does other music also have this problem?

**Comment by niuyaweiqq** [comment]
> Alright, can you try this?
> 
> * Export database from settings
> * Clear app data
> * Import database, but when it asks to restore settings, say no. Restoring settings will also restore cookies and we don't want that.
> * Try that URL again.
> 
> After you're done testing, you can clear app data and import the database along with settings to get back to where you started.

The problem still exists. I tried to switch VPN downloads in other countries without any problems. My question is whether this music is forbidden in France, if it is forbidden, why can webview be played, and if it is not forbidden in France, why is there only a French VPN Can't download the webpage. Does other music also have this problem?

**Comment by opusforlife2** [comment]
Could you install a debug apk (for example from the pinned issue), connect your phone to your PC through USB, open Chrome, go to chrome://inspect, open that debug version of NewPipe on your phone, select that NewPipe instance in Chrome, go to the Network tab, try to reproduce the issue, and once you've successfully reproduced the issue, right-click in Chrome, select "Save as HAR with Content" and send us that HAR?

**Comment by niuyaweiqq** [comment]
> Could you install a debug apk (for example from the pinned issue), connect your phone to your PC through USB, open Chrome, go to chrome://inspect, open that debug version of NewPipe on your phone, select that NewPipe instance in Chrome, go to the Network tab, try to reproduce the issue, and once you've successfully reproduced the issue, right-click in Chrome, select "Save as HAR with Content" and send us that HAR?

[m.youtube.com-1600352597717.log](https://github.com/TeamNewPipe/NewPipe/files/5239784/m.youtube.com-1600352597717.log)
Is it this type of file?

**Comment by niuyaweiqq** [comment]
> Could you install a debug apk (for example from the pinned issue), connect your phone to your PC through USB, open Chrome, go to chrome://inspect, open that debug version of NewPipe on your phone, select that NewPipe instance in Chrome, go to the Network tab, try to reproduce the issue, and once you've successfully reproduced the issue, right-click in Chrome, select "Save as HAR with Content" and send us that HAR?





> Could you install a debug apk (for example from the pinned issue), connect your phone to your PC through USB, open Chrome, go to chrome://inspect, open that debug version of NewPipe on your phone, select that NewPipe instance in Chrome, go to the Network tab, try to reproduce the issue, and once you've successfully reproduced the issue, right-click in Chrome, select "Save as HAR with Content" and send us that HAR?

I completed the error collection according to the operation you said. The following is the file you want, but this problem seems to be a robot verification problem, but after the NewPipe test version is entered, no video can be opened, all prompting me to need robot verification, but I can browse the video normally with the local webview, but I can’t download the webpage. I don’t know why this is. I don’t know if you can answer my doubts. Thanks again.
[ssssfdd.har.log](https://github.com/TeamNewPipe/NewPipe/files/5244040/ssssfdd.har.log)

**Comment by opusforlife2** [comment]
Thanks! Yeah, the previous one doesn't look like a HAR. This one does. Now devs can check it out.

**Comment by niuyaweiqq** [comment]
> Thanks! Yeah, the previous one doesn't look like a HAR. This one does. Now devs can check it out.

I’m very sorry, I studied it for a while before I figured it out. I opened it in Log format and then it became .log. I don’t know if it will affect it. And I found a problem, that is, if you use the search that comes with Youtube Find out phpGv04wx5c this music can be played directly, but if I open it with the address https://m.youtube.com/watch?v=phpGv04wx5c, it will prompt me that I need robot verification. This is a phenomenon I have seen, I hope it will be useful to you help.
